third_party/README.md

```diff
@@ -0,0 +1,2 @@
+See `./install.sh` for information on where the `*.js` files in this directory
```

Comment:
`third_party/` contains imported code. We could also depend on lighthouse itself, but doesn't seem quite worth it. (And would still need to patch `manifest-parser.js`, I think.